src/lib/libstack_trace.js

```diff
@@ -101,29 +101,29 @@ var LibraryStackTrace = {
   $convertFrameToPC__docs: '/** @returns {number} */',
   $convertFrameToPC__internal: true,
   $convertFrameToPC: (frame) => {
-#if !USE_OFFSET_CONVERTER
-    abort('Cannot use convertFrameToPC (needed by __builtin_return_address) without -sUSE_OFFSET_CONVERTER');
-#else
-#if ASSERTIONS
-    assert(wasmOffsetConverter, 'wasmOffsetConverter global not found');
-#endif
     var match;
 
     if (match = /\bwasm-function\[\d+\]:(0x[0-9a-f]+)/.exec(frame)) {
       // some engines give the binary offset directly, so we use that as return address
       return +match[1];
     } else if (match = /\bwasm-function\[(\d+)\]:(\d+)/.exec(frame)) {
-      // other engines only give function index and offset in the function,
+      // Older versions of v8 give function index and offset in the function,
       // so we try using the offset converter. If that doesn't work,
       // we pack index and offset into a "return address"
+#if !USE_OFFSET_CONVERTER
+      abort('Legacy backtrace format detected but -sUSE_OFFSET_CONVERTER not present.')
+#else
+#if ASSERTIONS
+      assert(wasmOffsetConverter, 'wasmOffsetConverter global not found');
+#endif
       return wasmOffsetConverter.convert(+match[1], +match[2]);
```

Comment:
```suggestion
      return wasmOffsetConverter.convert(+match[1], +match[2]);
#endif
```
May as well only emit it when it is enabled.

Reply:
That is the case already... this code is already part of the #else block right?

Reply:
Oh sorry, I missed the if for ASSERTIONS... lgtm as is